### ISSUE TITLE
Replace ACTION_MEDIA_MOUNTED broadcast that seems protected since Kitkat

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#2280]
```
Permission Denial: not allowed to send broadcast android.intent.action.MEDIA_MOUNTED from pid=13367, uid=10242
14 – StackTrace: java.lang.SecurityException: Permission Denial: not allowed to send broadcast android.intent.action.MEDIA_MOUNTED from pid=13367, uid=10242
```

**Comment by maxme** [comment]
Probably related: https://stackoverflow.com/questions/24072489/java-lang-securityexception-permission-denial-not-allowed-to-send-broadcast-an

**Comment by tonyr59h** [comment]
Fixed in #2488, closing.
